tests/ikev2-weak-dh/test.yaml

```diff
@@ -3,6 +3,7 @@ requires:
     - HAVE_LIBJANSSON
     - RUST
   min-version: 4.1.0
+  lt-version: 6
```

Comment:
@jasonish this test is getting deprecated and replaced by `ike-weak-dh`
Is this the way to go ?

@fhonza could you modify ikev2-weak-dh` README (or add a comment in this yaml) to explain this ?

Reply:
with the unification of ikev1 and ikev2 into ike both parsers now use output-json-ike.c, which just adds the event_type "ike" (if i've debugged correctly), which is breaking the old test. Perhaps i can try to use either ikev1 or ikev2 depending on the protocol version.
In this function (JsonIKELogger(...)) I have the state as a parameter, perhaps it's possible to check there.

In this case the existing test could be left untouched, would be better?

Reply:
I think event_type "ike" is good.
@jasonish what do you think ?

Reply:
Just "ike" would be idea I think.

Reply:
Does the framework enable to have some checks with some version and some others checks with other versions ?
cc @inashivb 

That is what we want here I think.
The pcap is the same.
Previously, we checked `ikev2` field in eve.json, and now we will want it to be simply `ike`

Reply:
Thanks Shivani.
Could you do this @fhonza ?

Reply:
great! created a new PR...